src/iregnet_fit.cpp

```diff
@@ -325,13 +325,18 @@ fit_cpp(Rcpp::NumericMatrix X, Rcpp::NumericMatrix y,
     if (n_iters[m] == max_iter)
       error_status = -1;
     if (std::isinf(out_loglik[m]))
-      error_status = -3;
+      error_status = -2;
```

Comment:
this is not being used. Kindly handle this.

Reply:
It is not being used but I had plans to use it afterwards when displaying the summary of fit. Nevertheless, it isn't important so I'll remove it.